static/configs/tslint_json

```diff
@@ -0,0 +1,129 @@
+{
+    "compilerOptions": {
+        "experimentalDecorators": true
+    },
+    "defaultSeverity": "error",
+    "extends": [
+        "tslint:recommended"
+    ],
+    "jsRules": {},
+    "rulesDirectory": [],
+    "rules": {
+        "ordered-imports": false,
+        "callable-types": true,
+        "class-name": true,
+        "comment-format": [
+            true,
+            "check-space"
+        ],
+        "curly": true,
+        "eofline": true,
+        "forin": true,
+        "import-spacing": true,
+        "indent": [
+            true,
+            "spaces",
+            4
+        ],
+        "interface-name": [
+            true,
+            "never-prefix"
+        ],
+        "interface-over-type-literal": true,
+        "label-position": true,
+        "max-line-length": [
+            true,
+            {
+                "limit": 140,
+                "ignore-pattern": "^import |^export {(.*?)}"
+            }
+        ],
+        "member-access": false,
+        "member-ordering": [
+            true,
+            {
+                "order": [
+                    "static-field",
+                    "instance-field",
+                    "static-method",
+                    "instance-method"
+                ]
+            }
+        ],
+        "no-arg": true,
+        "no-bitwise": true,
+        "no-console": [
+            true,
+            "debug",
+            "info",
+            "time",
+            "timeEnd",
+            "trace"
+        ],
+        "no-construct": true,
+        "no-debugger": true,
+        "no-duplicate-variable": true,
+        "no-empty": false,
+        "no-empty-interface": true,
+        "no-eval": true,
+        "no-inferrable-types": true,
+        "no-shadowed-variable": true,
+        "no-string-literal": false,
+        "no-string-throw": true,
+        "no-switch-case-fall-through": true,
+        "no-trailing-whitespace": true,
+        "no-unused-expression": true,
+        "no-var-keyword": true,
+        "object-literal-shorthand": [
+            true,
+            "never"
+        ],
+        "object-literal-sort-keys": false,
+        "no-floating-promises": true,
+        "one-line": [
+            true,
+            "check-open-brace",
+            "check-catch",
+            "check-whitespace"
+        ],
+        "prefer-const": false,
```

Comment:
I would suggest `true`

Reply:
Good catch, same as with styling, we have a task in Asana where we're reevaluating these things (and migrating to ESLint): https://app.asana.com/0/1199525026761269/1199525026761272/f

Reply:
I agreed.

Reply:
Can you add the change @Alex-DA ?

Reply:
Fixed in: https://github.com/mobilejazz/harmony-reference/pull/82/commits/f28676e90e3cb8ce06ee82fd4c4405a822d2c28a